test/test_loggers.py

```diff
@@ -287,3 +286,9 @@ def test_raw_trace_log_can_be_yaml_loaded(monkeypatch):
         assert rec3[time3]['offset'] == 34
     for filename in created_files:
         os.remove(filename)
+
+
```

Comment:
What this test is checking? It will always pass !!!
No asserts !!!
It should use mocking to check if files have been open in target folder (mocking to not use real file system)

Reply:
So I create a dummy handler use mock to pass it into reconfigure method, and Mock all method which can use real file system. Is it ok?

Reply:
Ok I add mocking to not use real file system and check that new file_path is correct.